client/src/app/demande-feedback/demande-feedback.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="container-fluid">
+    <div class="row justify-content-center padTop">
+        <p class="col-6">Demande un feedback à un(e) collégue</p>
+    </div>
+    <div class="row justify-content-center padTop">
+        <form (submit)="onSubmitRequest()" id="askFeedback" #askFeedback="ngForm" novalidate>
+            <div class="form-group">
+                <label class="col-2">Votre Email:</label>
+                <input type="text" id="email" name="email" [(ngModel)]="askFeedbackRequest.email"  class="col-6 form-control required" #email="ngModel" required placeholder="Ex: pierre.pompidor@gmail.com"/>
+            </div>
+            <div class="form-group padTop">
+                <label class="col-2">Collégue Email:</label>
+                <input  type="text" id="senderEmail" name="senderEmail"  [(ngModel)]="askFeedbackRequest.senderEmail" required class="col-6 form-control required" #senderEmail="ngModel" placeholder="Ex: marie.mettrand@gmail.com"/>
```

Comment:
```suggestion
                <input  type="text" id="senderEmail" name="senderEmail"  [(ngModel)]="askFeedbackRequest.senderEmail" required class="col-6 form-control required" #senderEmail="ngModel" placeholder="Ex: marie.mettrand@example.com"/>
```

C'est pas une bonne pratique de mettre un vrai email, on sait jamais si cette email n'existe pas !

Reply:
Je pensais puisque c'est un example précis on doit mettre un domain précis, d'accord je change